datajoint/schemas.py

```diff
@@ -171,7 +171,7 @@ def __call__(self, cls, *, context=None):
         context = context or self.context or inspect.currentframe().f_back.f_locals
         if issubclass(cls, Part):
             raise DataJointError(
-                "The schema decorator should not be applied to Part relations"
+                "The schema decorator should not be applied to Part tables."
```

Comment:
There are about 25 uses of the word "relation" across the codebase. Please replace them all with either "table" or "query" as appropriate.

Reply:
Thanks, I will get that done today.